### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by misiek08** [daniel0916/MCProxy#3]
I see it's nice project, but can you provide any documentation?

**Comment by daniel0916** [comment]
No, there isn't currently any documentation und there is also not a plugin api. I have plans for a big update and maybe i will find time for this.

**Comment by misiek08** [comment]
To make it clear - update in few days or you have other things to do and that's just hobby project? I'm looking for BungeeCord alternative and still can't find anything reliable.

**Comment by LO1ZB** [comment]
where do you need help with?
compiling/installing or configuration?

**Comment by misiek08** [comment]
For now - configuration.

How far is it for being basic compatible with BC? I mean plugin message channel to redirect user or sending real user IP in packet?

**Comment by daniel0916** [comment]
Here a little explanation:
Plugin message Channel: MCProxy
When you send "connect Server5". The proxy will search in the config for the ip and will move you to this server.
When you send "get uuid". You will become the UUID of the player.

And the /server Command.

More isn't available currently. 

It's a hobby project but i will work on updates etc. You can see in the 1.8-Development branch. I started for a longer time with a 1.8 update but it have many bugs. Currently i will start to work on a new method to modify the packets...

**Comment by daniel0916** [comment]
Currently i'm working on some new optimations. I made for this the development branch.
